backend/hitas/services/validation.py

```diff
@@ -0,0 +1,17 @@
+import re
+
+from rest_framework.exceptions import ValidationError
+
+
+def validate_postal_code(postal_code: str) -> str:
```

Comment:
We have already a bunch of `validate_` methods in `hitas/models/utils.py`, maybe those can be moved here as well (as a part of the general architecture improvements)

Reply:
Was planning to, although when I tried this some time ago there were some import problems, but I think those can be solved with some more effort.